sections/product-recommendations.liquid

```diff
@@ -56,9 +58,14 @@
             const html = document.createElement('div');
             html.innerHTML = text;
             const recommendations = html.querySelector('product-recommendations');
+
             if (recommendations && recommendations.innerHTML.trim().length) {
               this.innerHTML = recommendations.innerHTML;
             }
+
+            if (html.querySelectorAll('.grid__item').length > 0) {
```

Comment:
I believe we only need to check if there is at least 1 grid item, so we don't need to query all instances

```suggestion
            if (html.querySelector('.grid__item')) {
```

Reply:
True - good point!